index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
+    <link rel="stylesheet" href="index.css"/>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/js/bootstrap.bundle.min.js" integrity="sha384-w76AqPfDkMBDXo30jS1Sgez6pr3x5MlQ1ZAGC+nuZB+EYdgRZgiwxhTBTkF7CXvN" crossorigin="anonymous"></script>
+    <script src="https://kit.fontawesome.com/6587bf79be.js" crossorigin="anonymous"></script>
+    <title>Document</title>
+</head>
+<body>
+    <!-- <nav id="navbar" class="navbar">
+        <a href="#welcome-section">About</a>
+         <a href="#work">Work</a>
+          <a href="#contacts-link" id="contacts-link" target="_blank">Contacts</a>
+    </nav> -->
+    <nav id="navbar" class="navbar navbar-expand-lg ">
+        <div class="container-fluid">
+          <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+            <div class="navbar-nav">
+              <a class="nav-link" href="#">About</a>
+              <a class="nav-link" href="#">Work</a>
+              <a class="nav-link ">Contact</a>
+            </div>
+          </div>
+        </div>
+      </nav>
+      <div id="welcome-section">
+        <h1>Hi,I am Mimic</h1>
+        <p>a Web Developer</p>
+      </div>
+      <section id="projects">
+        <div class="project-tile">
+       <h2>These are some of my projects</h2>
+        </div>
+        <div id="project-link">
+         <div id="image-link">
+          <a  href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank"><img id="image" src="images/download.jpg" alt=""></a>
+          <a id="link" href="https://codepen.io/freeCodeCamp/full/zNqgVx"  target="_blank" >Tribute</a>
+         </div>
```

Comment:
Lovely work 👏🏻 on the container of the project-link by using however I suggest you please use ul instead of div because using ul over the div is semantically more correct. It would help your website to be more accessible to the audiences that have to rely on the screen-reader. Kindly make sure that all your child list or project-link cards also use li to make it perfect otherwise it will results again in bad practices.
```suggestion
         </div>
```